### ISSUE TITLE
create spectral invert effect

### ISSUE THREAD

**Issue by endphp2** [andrewreeman/SpectralSuite#2]
i saw this feature on a plugin years ago, it's cool. i'll send it if i found out.
and it's so related to spectral gate plugin
it inverts volume of FFTs. louders get mute and more quite sounds get loud

**Comment by andrewreeman** [comment]
Yes I think I understand this. So we could control the loud/medium/soft balances of the frequencies with three sliders perhaps?
